spec/pivotal-ui-react/tab/tab_spec.js

```diff
@@ -27,6 +33,26 @@ describe('Tabs', function() {
       expect('.tab-simple nav ul.nav.nav-tabs li.active').toContainText('Tab1');
       expect('.tab-simple .tab-content .tab-pane.fade.active.in').toContainText('Content1');
     });
+
+    itPropagatesAttributes('#root .tab-simple', {
+      className: 'testSimpleTabsClass',
+      id: 'testId',
+      style: {opacity: '0.5'}
+    });
+
+    describe('Tab', function() {
```

Comment:
We could use the `itPropagatesAttributes` helper here.
Also, why is the Tab describe block nested?